index.js

```diff
@@ -1,7 +1,7 @@
-const React = require('react')
-const ReactDOM = require('react-dom')
-const A11yDialog = require('a11y-dialog')
-const PropTypes = require('prop-types')
+import React from 'react'
```

Comment:
Uh, not sure where this is coming from. Might have been my local Prettier. I will fix.